Source/Core/VideoCommon/HiresTextures.h

```diff
@@ -40,6 +42,9 @@ class HiresTexture
 	std::vector<Level> m_levels;
 
 private:
+	static HiresTexture* Load(std::string base_filename, u32 width, u32 height);
```

Comment:
`const std::string&` (assuming I didn't misread the code and the parameter isn't modified in the function)